From 53a5664a61c744781291e39b3d9cd1ccb79dd667 Mon Sep 17 00:00:00 2001
From: Richard Zhu <hongxing.zhu@nxp.com>
Date: Tue, 22 May 2018 17:45:45 +0800
Subject: [PATCH 3882/5242] MLK-18386 ARM64: dts: imx8mm: fix pcie pll can't
 be locked in resume

commit  635048f75184676ebbd7e42a78fabc6e61239fbe from
https://source.codeaurora.org/external/imx/linux-imx.git

pcie aux clock is mandatory required by pcie power management.
add the aux clock into imx8mm pcie dts node explicitly.
pcie ctrl clock would be turned on, when pcie root clock
is enabled.

Signed-off-by: Richard Zhu <hongxing.zhu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
index 28775fe..c90049c 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
@@ -927,7 +927,7 @@
 				<0 0 0 3 &gic GIC_SPI 123 IRQ_TYPE_LEVEL_HIGH>,
 				<0 0 0 4 &gic GIC_SPI 122 IRQ_TYPE_LEVEL_HIGH>;
 		clocks = <&clk IMX8MM_CLK_PCIE1_ROOT>,
-			<&clk IMX8MM_CLK_PCIE1_CTRL_CG>,
+			<&clk IMX8MM_CLK_PCIE1_AUX_CG>,
 			<&clk IMX8MM_CLK_PCIE1_PHY_CG>;
 		clock-names = "pcie", "pcie_bus", "pcie_phy";
 		fsl,max-link-speed = <2>;
-- 
1.7.9.5

